CHANGELOG.md

```diff
@@ -4,6 +4,30 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## 0.10.0 (December 29th, 2019)
+
+The 0.10.0 makes some changes that will allow us to support both sync and async interfaces.
+
+In particular with streaming responses the `response.read()` method becomes `response.aread()`, and the `response.close()` method becomes `response.aclose()`.
+
+If following redirects explicitly the `response.next()` method becomes `response.anext()`.
+
+### Fixed
+
+- End HTTP/2 streams immediately on no-body requests, rather than sending an empty body message. (Pull #682)
```

Comment:
Should we add a mention to #666? I think it's a valuable enough UX improvement to be mentioned here.

```markdown
- Improve typing for `Response.request`: switch from `Optional[Request]` to `Request`. (Pull #666)
```

Reply:
Good call